articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md

```diff
@@ -88,10 +90,25 @@ The creation of product families from Dynamics 365 Sales isn't supported with th
 
 The synchronization of products happens from the finance and operations app to Dataverse. This means that the values of the product table columns can be changed in Dataverse, but when the synchronization is triggered (when a product column is modified in a finance and operations app), this will overwrite the values in Dataverse.
 
+### New approach for integration of products
+Starting from Supply Chain Management version 10.0.39 and Dynamics 365 Supply Chain Extended v2.3.4.294 a new approach is introduced for product synchronization to impvove the overall performance.
+In this approach, the new data entities [DV released distinct products](mapping-reference.md#242) and [DV released products](mapping-reference.md#243) only contain  the information that is synchronazied to Dataverse, which makes the data entities simpler, having fewer joints and data sources. And the dataverse plugins are taking care of updating the relevant product information from **msdyn\_sharedproductdetails** to **Product**.
+ 
+As it is mentioned in [Sync on-demand with the Supply Chain Management pricing engine](../../../fin-ops/data-entities/pricing-engine.md) Dynamics 365 Sales uses  Supply Chain Management pricing engine for price related calculations, so in this approach the UnitCost and SalesPrice are not being synchronized, which solves many isues with dual-write timeouts.
```

Comment:
we probably want to drop this part.

Reply:
So, not mentioning about UnitCost/SalesPrice at all?